federation-js/src/composition/validate/postComposition/__tests__/keyFieldsMissingOnBase.test.ts

```diff
@@ -46,6 +46,46 @@ describe('keyFieldsMissingOnBase', () => {
     expect(warnings).toHaveLength(0);
   });
 
+  it('returns no warnings with proper @key usage for provided field', () => {
+    const serviceA = {
+      // FIXME: add second key "upc" when duplicate directives are supported
+      // i.e. @key(fields: "sku") @key(fields: "upc")
```

Comment:
This is supported so probably either remove or implement the recommendation (I know this is just copied from above)